README.md

```diff
@@ -1 +1,544 @@
 # S4 - Server Side Swift Standards
+
+## Byte
+
+The `Byte` type represents a byte.
+
+```swift
+public typealias Byte = UInt8
+```
+
+### Motivation
+
+`Byte` is clearer than `UInt8` in the context of binary data.
+
+## Data
+
+The `Data` type represents binary data as a collection of bytes.
+
+> An alternative name could be `Buffer`.
+
+```swift
+public struct Data {
+    public var bytes: [Byte]
+}
+```
+
+### Motivation
+
+Having a type for binary data makes low level operations a lot easier. Specially when interfacing with `C`.
```

Comment:
I would avoid using language like `a lot` or even `easier`. Technical documentation should be technical. 
> Binary data types facilitate standardization of low level operations.